pkg/virt-launcher/virtwrap/agent-poller/agent_poller.go

```diff
@@ -367,64 +363,69 @@ func executeAgentCommands(commands []AgentCommand, con cli.Connection, agentStor
 		}
 
 		switch command {
-		case GET_FSFREEZE_STATUS:
+		case GetFSFreezeStatus:
 			fsfreezeStatus, err := ParseFSFreezeStatus(cmdResult)
 			if err != nil {
 				log.Log.Errorf("Cannot parse guest agent fsfreeze status %s", err.Error())
 				continue
 			}
-			agentStore.Store(GET_FSFREEZE_STATUS, fsfreezeStatus)
-		case GET_FILESYSTEM:
+			agentStore.Store(GetFSFreezeStatus, fsfreezeStatus)
+		case GetFilesystem:
 			filesystems, err := parseFilesystem(cmdResult)
 			if err != nil {
 				log.Log.Errorf("Cannot parse guest agent filesystem %s", err.Error())
 				continue
 			}
-			agentStore.Store(GET_FILESYSTEM, filesystems)
-		case GET_AGENT:
+			agentStore.Store(GetFilesystem, filesystems)
+		case GetAgent:
 			agent, err := parseAgent(cmdResult)
 			if err != nil {
 				log.Log.Errorf("Cannot parse guest agent information %s", err.Error())
 				continue
 			}
-			agentStore.Store(GET_AGENT, agent)
+			agentStore.Store(GetAgent, agent)
 		}
 	}
 }
 
-func fetchAndStoreGuestInfo(types libvirt.DomainGuestInfoTypes, con cli.Connection, agentStore *AsyncAgentStore, domainName string) {
-	log.Log.Infof("Polling API operations: %v", types)
+func fetchAndStoreGuestInfo(infoTypes libvirt.DomainGuestInfoTypes, con cli.Connection, agentStore *AsyncAgentStore, domainName string) {
+	log.Log.Infof("Polling API operations: %v", infoTypes)
 
 	domain, err := con.LookupDomainByName(domainName)
 	if err != nil {
 		log.Log.Errorf("Domain lookup failed: %v", err)
 		return
 	}
-	defer domain.Free()
 
-	guestInfo, err := domain.GetGuestInfo(types, 0)
+	defer func() {
+		if freeErr := domain.Free(); freeErr != nil {
```

Comment:
It's safe to ignore the error here. The `domain.Free()` function would return an error only if it's called multiple times, of if the domain object is invalid.
Neither condition is true in this case.

Reply:
Okay let's keep it ignored. Done.